src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1651,6 +1651,38 @@ GlobeControls.prototype.getScale = function getScale(pitch) {
 };
 
 /**
+ * Length in unit meter or kilometer
+ * @typedef {Object} Length
+ * @property {number} value distance value
+ * @property {string} unit distance unit
+ */
+
+/**
+ * Convert pixel in length (meter or kilometer)
+ * @param      {number}  pixel   The pixel
+ * @param      {number}  pitch   The pitch
+ * @return     {Length} length
```

Comment:
This documentation is wrong (`meter or kilometer`) and a typical example of why structured / linted documentation easily leads to useless documentation (ping issue #314).
Sure all fields are documented, have their type specified but how can `@param      {number}  pixel   The pixel`  be useful to anyone? 
Or `The pitch`??